libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -44,117 +44,234 @@ export type TestActivityInspector = (activity: Partial<Activity>, description: s
  */
 export class TestAdapter extends BotAdapter implements ExtendedUserTokenProvider {
     /**
-     * @private
-     * INTERNAL: used to drive the promise chain forward when running tests.
+     * Creates a new TestAdapter instance.
+     * @param logicOrConversation The bots logic that's under test.
+     * @param template (Optional) activity containing default values to assign to all test messages received.
      */
-    public readonly activityBuffer: Partial<Activity>[] = [];
```

Comment:
>activityBuffer [](start = 20, length = 14)

This could be perceived as a breaking change.

Reply:
This deletion/rename from `activityBuffer` to `activityQueue` seems to be intended as a way to achieve parity with .NET. 

We have to decide the level of "breaking" changes we want to take and migration pattern that we want to dictate to customers if we decide to rename these public properties and methods.

Reply:
alias `activityBuffer` to `activeQueue` to keep backward compatibity, which may also reduce the number of files touched.